### ISSUE TITLE
Learning rate

### ISSUE THREAD

**Issue by sahiliitm** [atgambardella/pytorch-es#5]
Is the default learning rate of 0.3 corresponding to 12 threads or 40?
Should learning rate scale linearly with batch size?

**Comment by atgambardella** [comment]
The defaults are assuming that you're training on cartpole, with 40 processes (that being said, most reasonable hyperparameters should work with cartpole). Learning rate should scale in some way with batch size, but most likely not linearly. You'll just have to experiment with what works.

I'll also sometime soon be adding other optimizers, so learning rates will naturally be different for them anyway.
